R/http.R

```diff
@@ -119,6 +116,12 @@ TWIT_paginate_max_id <- function(token, api, params,
                                  count_param = "count", 
                                  retryonratelimit = FALSE,
                                  verbose = TRUE) {
+  if (!is.null(max_id)) {
+    max_id <- rtweet::max_id(max_id)  
+  }
+  if (!is.null(since_id)) {
+    since_id <- rtweet::since_id(since_id)  
```

Comment:
Why the rtweet::max_id and rtweet::since_id? Isn't max_id and since_id enough?

Reply:
I think it makes the code a little easier to follow than `since_id <- since_id(since_id)`